lib/flood_risk_engine/engine.rb

```diff
@@ -12,12 +12,35 @@ class Engine < ::Rails::Engine
       app.config.i18n.load_path += Dir[config.root.join("config/locales/**/**/", "*.{rb,yml}").to_s]
     end
 
+    # Automatically add our migrations into the main apps migrations
```

Comment:
Can the comment be updated to reflect the current context. We don't have a 'core' in this project. Also does the rest of the comment still apply? In which case should this be a thing to discuss in the dev mtg?